utilities/debug.cc

```diff
@@ -14,16 +14,25 @@ namespace rocksdb {
 Status GetAllKeyVersions(DB* db, Slice begin_key, Slice end_key,
                          size_t max_num_ikeys,
                          std::vector<KeyVersion>* key_versions) {
+  assert(nullptr != db);
```

Comment:
This is a user input. Maybe return an error instead of asserting.

Reply:
Done.